src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <p>
+          {' Posted by '}
+          <UserInfo user={user} />
```

Comment:
find can return undefined, so better to add the condition here to render it
```suggestion
          {user && <UserInfo user={user} />}
```